robot/Cumulus/resources/EngagementPlanTemplatesPageObject.py

```diff
@@ -7,24 +7,40 @@
 @pageobject("Listing", "Engagement_Plan_Template__c")
 class EngagementPlanListPage(BaseNPSPPage, ListingPage):
 
-    
     def _is_current_page(self):
         """
-        Waits for the current page to be a Data Import list view
+        Waits for the current page to be a Engagement Plan list view page
         """
-        self.selenium.wait_until_location_contains("/list",timeout=60, message="Records list view did not load in 1 min")
+        self.selenium.wait_until_location_contains("/list",timeout=60, message="Engagement plan list view did not load in 1 min")
         self.selenium.location_should_contain("Engagement_Plan_Template__c",message="Current page is not an Engagement Plan List view")
-            
+
+    def go_to_engagement_plan_page(self,action,engagementid=None):
+        """ Navigates to the specified mode of engagement plant template page """
+        url = self.cumulusci.org.lightning_base_url
```

Comment:
Is this a base url? if so can you please rename it as you have one other url mentioned below so it looks confusing as to which url we are referring to here.

Reply:
took care of it in the new keyword added